sal_interfaces/ATAOS/ATAOS_Events.xml

```diff
@@ -442,4 +449,47 @@
         <Count>1</Count>
     </item>
 </SALEvent>
+<SALEvent>
+    <Subsystem>ATAOS</Subsystem>
+    <Version>3.8</Version>
+    <Author>Tiago Ribeiro</Author>
+    <EFDB_Topic>ATAOS_logevent_focusOffsetSummary</EFDB_Topic>
+    <Alias>focusOffsetSummary</Alias>
+    <Explanation>http://sal.lsst.org</Explanation>
+    <item>
+        <EFDB_Name>totalFocusCorrectionOffset</EFDB_Name>
```

Comment:
So, the name of the topic is already `focusOffsetSummary`, I think the attributes could have simpler names. I would suggest:

`totalFocusCorrectionOffset` -> `total`
`cumulativeUserOffset` -> `user`
`filterOffset` -> `filter`
`disperserOffset` -> `disperser`
`wavelengthOffset` -> `wavelength`

Reply:
renaming completed. Note that this affected both the XML as well as the test_csc and ataos_csc files, which are being modified under the same ticket.